Workbooks/AKS/Node GPU/Node GPU.workbook

```diff
@@ -97,25 +98,27 @@
             "name": "clusterId",
             "type": 1,
             "description": "Filter the selected workspace by cluster id",
-            "isHiddenWhenLocked": true
+            "isHiddenWhenLocked": true,
+            "value": "/subscriptions/692aea0b-2d89-4e7e-ae30-fffe40782ee2/resourceGroups/nickb-rg-test/providers/Microsoft.ContainerService/managedClusters/nickb-aks-cluster-banner-test",
+            "resourceType": "microsoft.insights/components"
           },
           {
             "id": "51f0a81e-6eef-4129-b8a4-dc7dd4114aff",
             "version": "KqlParameterItem/1.0",
             "name": "clusterIdWhereClause",
             "type": 1,
             "description": "use this to filter by clusterid ",
-            "value": "| where \"a\" == \"a\"",
+            "value": "| where clusterId =~ \"/subscriptions/692aea0b-2d89-4e7e-ae30-fffe40782ee2/resourceGroups/nickb-rg-test/providers/Microsoft.ContainerService/managedClusters/nickb-aks-cluster-banner-test\"",
```

Comment:
this is going to fail to validate since the template has resource ids hardcoded in it

Reply:
Yea I loaded the workbook, made some edits, then copied the JSON...Because values are loaded into the workbook template, the editing experience is difficult and frustrating. Instead of going through the hassle of looking through the JSON and making ad hoc fixes the second time around, I just identified how my changes manifested in the JSON, then copied the JSON from github, and edited it according to the changes I saw in the JSON from my manual editing of the workbook...Then I loaded that JSON into the workbook to make sure I didn't mess things up. Vishwa brought up to Austin how it'd be great if there was an export button that stripped the values that were loaded into the workbook template, as well as the parameters that were changed while using the workbook. Obviously there is some nuance involved here, but hopefully you get the gist. I would love to see a feature like this in the future.